.changelogs/9064.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Fixed maximum call stack size error in indexMapper by introducing the new `getNearestNotHiddenIndex` method.",
```

Comment:
```suggestion
  "title": "Fixed an issue where calling `getFirstNotHiddenIndex()` on a Handsontable instance with more than 6000 hidden rows or columns resulted in an uncaught `RangeError`, by adding a new method: `getNearestNotHiddenIndex()`. The `getFirstNotHiddenIndex()` method still works, but is marked as deprecated.
```

Reply:
@budnix – from what I see in the in the PR's description, we could also present it as "Added", like this:

```
  "title": "Added `getNearestNotHiddenIndex()`, a new method that finds a visual index of the nearest not-hidden row or column, and works even with large numbers of hidden rows or columns. The previous method, `getFirstNotHiddenIndex()` still works, but is marked as deprecated.
```

Reply:
Thanks. I've created a new [PR](https://github.com/handsontable/handsontable/pull/9519) with your suggestions. I completely forgot to wait for your review 🤦

Reply:
> I completely forgot to wait for your review 🤦

You shouldn't wait, as we haven't changed the process. But it is completely fine and expected that @kirszenbaum reviews the docs asynchronously to the PR.